front-page.php

```diff
@@ -10,25 +10,20 @@
 <?php
 
 // WP_Query arguments
-$args = array (
-  'post_type'              => array( 'project' ),
-  'posts_per_page'         => '5',
-);
+$projects = get_post_meta(get_the_ID(), '_igv_front_projects', true);
 
-// The Query
-$query = new WP_Query( $args );
+//pr($projects); die;
 
 // The Loop
-if ( $query->have_posts() ) {
+if ($projects) {
```

Comment:
you can preserve the WP_Query here but just adding the `post__in` argument for the query from an array of the returned ids. Then the rest of the code can stay the same